src/warnet/deploy.py

```diff
@@ -42,13 +49,31 @@ def validate_directory(ctx, param, value):
     callback=validate_directory,
 )
 @click.option("--debug", is_flag=True)
-def deploy(directory, debug):
+@click.option("--namespace", type=str, help="Specify a namespace in which to deploy the network")
+@click.option("--to-all-users", is_flag=True, help="Deploy network to all user namespaces")
```

Comment:
I'm not crazy about this option.. it feels like we are building a use case specific convenience flag into the CLI. What's the advantage of having this vs just writing a simple bash script that deploys the network to each namespace by calling `warnet deploy /path/to/network --namespace X; warnet deploy path/to/network --namespace Y` ?

Reply:
I also quite like that `warnet deploy` is somewhat generic in that it can deploy namespaces / networks. but now this flag is specific to only deploying networks.

Reply:
i like `--to-all-users` personally